src/libraries/System.Runtime/tests/System/DecimalTests.cs

```diff
@@ -619,6 +627,23 @@ public static void GetBits(decimal input, int[] expected)
             Assert.Equal(input, newValue);
         }
 
+        [Theory]
+        [MemberData(nameof(GetBits_TestData))]
+        public static void GetBitsSpan(decimal input, int[] expected)
+        {
+            Span<int> bits = new int[4];
+            int bitsWritten = decimal.GetBits(input, bits);
```

Comment:
No love for `TryGetBits` in the unit tests? Or a `GetBits` with a too-short array? :)

Reply:
Oops.  Added.  Thanks.